### PR TITLE
First try at making tests pass via GHA

### PR DESCRIPTION
**Pull request by berliner**

### CODE REVIEW

**Comment by berliner** [comment]
@cafuego Do you want to have a look at this before I merge this? I tried not to break anything, but you know how it is ...
Also, do you know why this PR is still waiting for travis?

File: .github/tests/test.sh
Comment (cafuego) [inline .github/tests/test.sh:1]: Andy has been doing some work with a (docker based) tool called `act` that allows you to run the actual github yaml locally. It adds a few env var based conditions so the comment and slack posting gets skipped. But that can be for a refactor.

File: .github/workflows/run-tests.yml [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+      - name: Software versions
+        id: versions
+        uses: cafuego/command-output@main
+        with:
+          run: |
+            php --version && composer --version
+
+      - name: Checkout Code
+        id: checkout
+        uses: actions/checkout@v3
```
Comment (cafuego) [inline .github/workflows/run-tests.yml:27]: I have a shiny new thing for that PHP version.

```suggestion
      - name: Checkout Code
        id: checkout
        uses: actions/checkout@v3
        
      - name: Extract PHP Version
        id: php
        uses: docker://ghcr.io/un-ocha/actions:extract-php-version-main
        with:
          docker_file: 'docker/Dockerfile'
          docker_image: 'public.ecr.aws/unocha/php-k8s'
          
      - name: Setup PHP with PECL extension
        uses: shivammathur/setup-php@v2
        with:
          php-version: ${{ steps.php.outputs.php_version }}
          tools: composer
        env:
          fail-fast: true

      - name: Software versions
        id: versions
        uses: cafuego/command-output@main
        with:
          run: |
            php --version && composer --version


```

File: .github/workflows/run-tests.yml [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+        with:
+          run: |
+            test ! -d ./html/modules/custom || find -L ./html/modules/custom -iregex '.*\.\(php\|module\|inc\|install\)$' -print0 | xargs -0 -n 1 -P 4 php -l
+            test ! -d ./html/themes/custom || find -L ./html/themes/custom -iregex '.*\.\(php\|theme\)$' -print0 | xargs -0 -n 1 -P 4 php -l
+        env:
+          fail-fast: true
+
+      - name: Configure AWS Credentials
+        id: aws
+        uses: aws-actions/configure-aws-credentials@v1-node16
```
Comment (cafuego) [inline .github/workflows/run-tests.yml:50]: The node version is apparently fixed in v2.
```suggestion
        uses: aws-actions/configure-aws-credentials@v2
```

File: .github/workflows/run-tests.yml [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+            }
+        env:
+          SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
+
+      - name: Slack Failure Notification
+        id: slack_failure
+        if: failure()
+        uses: slackapi/slack-github-action@v1.23.0
+        with:
+          channel-id: '${{ secrets.SLACK_CHANNEL }}'
```
Comment (cafuego) [inline .github/workflows/run-tests.yml:218]:
```suggestion
          channel-id: '${{ vars.SLACK_CHANNEL }}'
```

File: .github/workflows/run-tests.yml [hunk 4]
```diff
@@ -0,0 +1,238 @@
...
+
+            *Pusher: @${{ github.actor }}, Action: `${{ github.event_name }}`, Workflow: `${{ github.workflow }}`*
+          edit-mode: replace
+
+      - name: Slack Success Notification
+        id: slack_success
+        if: success()
+        uses: slackapi/slack-github-action@v1.23.0
+        with:
+          channel-id: '${{ secrets.SLACK_CHANNEL }}'
```
Comment (cafuego) [inline .github/workflows/run-tests.yml:191]: I'm slowly replacing this, so I have a non-secret value I can actually see in the UI :-)

```suggestion
          channel-id: '${{ vars.SLACK_CHANNEL }}'
```

**Review by cafuego — APPROVED** [review]
Approved with comments.

The tests are waiting for Travis because it's configured as a requirement in the repo branch protection settings, I'll turn that off.

**Comment by berliner** [comment]
Great review @cafuego! Thanks!

I start feeling that I should have asked you to review the same type of changes I also did for FTS and HPC Content a while back :/

**Comment by cafuego** [comment]
You can still ask me to look at those :-)

**Comment by berliner** [comment]
Would you? 👇 

https://github.com/UN-OCHA/ghi-site/blob/develop/.github/workflows/run-tests.yml
https://github.com/UN-OCHA/hpc-content-site/blob/develop/.github/workflows/run-tests.yml

**Comment by cafuego** [comment]
Of course, it's literally my job.

**Comment by berliner** [comment]
Great then, tell me what is wrong. It's all working to my satisfaction actually, but I'm sure things need to be updated or something ;)

**Comment by cafuego** [comment]
Yeah, they're all based on the previous incarnation, without PHP version detection or act support. But otherwise they look fine.

**Comment by berliner** [comment]
All merged now. Thanks!

**Pull request merged**